frontends/web/src/routes/account/account.tsx

```diff
@@ -186,11 +186,10 @@ export function Account({
     }
   }
 
+  const isAccountEmpty = balance && !balance.hasAvailable && !balance.hasIncoming && transactions && transactions.length === 0;
+
   const showBuyButton = moonpayBuySupported
-    && balance
-    && !balance.hasAvailable
-    && !balance.hasIncoming
-    && transactions && transactions.length === 0;
+    && isAccountEmpty;
```

Comment:
nit: it would help me for readability when the `isAccountEmpty` with many conditions would keep using newlines.

and the `showBuyButton ` with only `moonpayBuySupported && isAccountEmpty ` is fine on the same line.

Reply:
I assume with `staging-pocket` merged, there needs to be also a pocketBuySupprted similar to `moonpayBuySupported`.